.travis.yml

```diff
@@ -16,6 +16,7 @@ matrix:
         on:
           branch: master
 script:
+  - cargo fmt --all -- --check
```

Comment:
For this to work you also need to actually install `rustfmt` via `rustup` first.

Thanks!

Reply:
Now it only fails because 1.28 `rustfmt` was not stable yet and has slightly different output. I'd suggest to only run `rustfmt` for the `stable` build, none of the others.

Reply:
Totally agree. I'll tweak the check I just added